src/fluree/json_ld/impl/context.cljc

```diff
@@ -16,6 +16,17 @@
     (keyword (subs at-value 1))
     at-value))
 
+(defn vocab
+  "Computes vocab prefix given provided base-context, context, & vocab"
+  [base-context context vocab]
+  (let [base (or (get context "@base")
+                 (get base-context :base))]
+    (if (= "" vocab) ; empty string means use @base as @vocab
+      (when base (iri/add-trailing-slash base))
+      (if (and base (not (iri/absolute? vocab)))
+        ;; @vocab relative to @base
+        (iri/join base vocab)
+        (iri/add-trailing-slash vocab)))))
```

Comment:
I'm not sure we should be adding trailing slashes here if they aren't present in the original. At least, the json-ld playground does not add them, and I tend to refer to that for guidance.

Reply:
That was happening prior to this PR in the code I extracted out into this fn. So it's arguably outside the scope. But I'm happy to change it now if that's the consensus.

Reply:
Oh, in that case I'll see if I can get it to behave differently than the playground and open a separate issue if so.